pkgs/applications/networking/syncthing/default.nix

```diff
@@ -25,15 +25,28 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/etc/systemd/{system,user}
+
     cp bin/* $out/bin
+
+    substitute etc/linux-systemd/system/syncthing-resume.service \
+               $out/etc/systemd/system/syncthing-resume.service \
+               --replace /usr/bin/pkill ${procps}/bin/pkill
```

Comment:
Yes, `procfs` is not cross-platform. It should be there on Solaris and FreeBSD but not on macOS/OS X.